pygments/lexers/apdlexer.py

```diff
@@ -0,0 +1,211 @@
+import re
+from pygments.lexer import RegexLexer, bygroups, include
+from pygments.token import *
+
+__all__ = ['apdlexer']
+
+
+class apdlexer(RegexLexer):
+    name = 'ANSYS parametric design language'
```

Comment:
Please add documentation for this lexer here as a docstring, including a link to the language (if possible) and the versionadded macro

Reply:
Thanks. Changed.